presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -210,6 +211,10 @@ void PrestoServer::run() {
     exit(EXIT_FAILURE);
   }
 
+  if (systemConfig->enableRuntimeStatsCollection()) {
```

Comment:
@karteekmurthys Karteek, let me add this config in a separte PR. Inside Meta there is a small wrapper on PrestoServer that does following:

```
if (gflag enabled):
   initialize stats reporting
initialize presto server
```
I think we can use the config and remove the internal flag altogether. Let me introduce this flag in a PR and modify internal code. You can rebase on top of that.

Reply:
@karteekmurthys Here is the small PR
https://github.com/prestodb/presto/pull/21745
Once we merge it, I will refactor internal logic to use the cofnig property.